### PR TITLE
bug #74 Default font size should be

### PR DESCRIPTION
**Pull request by IndianBoyJii**
The « getter » of fontSize was not called, because the wrong attribute
(._book) was calling it. We added a new attribute (._bookViewer) to the
object ChapterContents, so then it can call the right function.

### CODE REVIEW

File: js/bookviewer.js [hunk 1]
```diff
@@ -61,7 +61,8 @@ function BookViewer(element) {
    *
    * @type {string}
    */
-  this._fontSize = Config.fontSize;
+  this._fontSize = Config.BookViewer.fontSize;
+  console.log("instanciation this._fontSize", this._fontSize);
```
Comment (Yoric) [inline js/bookviewer.js]: This call to `console.log` doesn't match the others. Either remove this call to `console.log` or make it look like

``` js
console.log("BookViewer", "constructor", "fontSize", this._fontSize);
```

File: js/bookviewer.js [hunk 2]
```diff
@@ -115,9 +116,10 @@ BookViewer.prototype.view = function(book, chapter = 0, endOfChapter = false) {
  *
  * @type {string} The font size, as a CSS property.
  */
+
```
Comment (Yoric) [inline js/bookviewer.js]: Nit: Please don't add unnecessary whitespace.

File: js/bookviewer.js [hunk 3]
```diff
@@ -115,9 +116,10 @@ BookViewer.prototype.view = function(book, chapter = 0, endOfChapter = false) {
...
 Object.defineProperty(BookViewer.prototype, "fontSize", {
-  get: function() {
-    return this._fontSize;
+  get: function() {
```
Comment (Yoric) [inline js/bookviewer.js]: Nit: Please remove the whitespace at the end of the line (here and the line below).

File: js/bookviewer.js [hunk 4]
```diff
@@ -392,7 +395,7 @@ BookViewer.prototype._keyboardNavigation = function(code) {
  * of contents.
  * @param {Book} book The book containing the chapter.
  */
-function ChapterContents(entry, num, book) {
+function ChapterContents(entry, num, book, bookViewer) {
```
Comment (Yoric) [inline js/bookviewer.js:396]: You added an argument, please add the documentation.

**Comment by Yoric** [comment]
Almost good. Can you fix the stuff above, so that I can merge?

**Comment by IndianBoyJii** [comment]
I fixed those stuff :)

**Comment by Yoric** [comment]
Almost good. Now, could you fold your patches into one?
